### PR TITLE
Add GPU support page

### PR DESCRIPTION
**Pull request by evilnick**

### CODE REVIEW

File: pages/k8s/gpu-workers.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+### Deploying CDK with GPU workers on AWS
+
+Using the recommended install method detailed in the [quickstart
+documentation][quickstart], you can select "CDK with NVIDIA GPU workers" at the
+first step, if you are using AWS. All the workers will then be deployed to
+machines with GPUs enabled. There is nothing further to do, enjoy!
+
+If you are installing CDK using a bundle, you can use constraints to specify
+that the worker units are deployed on GPU enabled machines. Because GPU support
```
Comment (tvansteenburgh) [inline pages/k8s/gpu-workers.md]: GPU-enabled

File: pages/k8s/gpu-workers.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+As demonstrated here, you can use multiple overlay files when deploying, so you
+can combine GPU support with an integrator charm or other custom configuration.
+
+You may then want to [test a GPU workload](#test)
+
+### Adding GPU workers with AWS
+
+It isn't necessary for all the worker units to have GPU support. You can simply
+add GPU enabled workers to a running cluster. The recommended way to do this is
```
Comment (tvansteenburgh) [inline pages/k8s/gpu-workers.md]: GPU-enabled

File: pages/k8s/gpu-workers.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+#gpu-overlay.yaml
+applications:
+  kubernetes-worker:
+    constraints: instance-type=p2.xlarge
+```
+
+And then deployed with CDK like this:
+
+```bash
+juju deploy charmed-kubernetes --overlay ~/path/aws-overlay.yaml --overlay ~/path/gpu-overlay.yaml
```
Comment (tvansteenburgh) [inline pages/k8s/gpu-workers.md]: canonical-kubernetes (we don't have a bundle named charmed-kubernetes, at least not yet)

Comment (evilnick) [reply pages/k8s/gpu-workers.md]: gah! I knew that. slip of the brain

**Pull request merged**